src/component/FeatureRenderer.js

```diff
@@ -238,106 +242,122 @@ Ext.define('GeoExt.component.FeatureRenderer', {
                 })
             ]
         });
-        var feature = this.getFeature();
+
         if (!feature) {
-            this.setFeature(this['get' + this.getSymbolType() + 'Feature']());
+            this.setFeature(this["get" + this.getSymbolType() + "Feature"]());
         } else {
             this.applyFeature(feature);
         }
         me.callParent(arguments);
     },
+
     /**
      * Draw the feature when we are rendered.
      *
      * @private
      */
-    onRender: function() {
+    onRender: function () {
         this.callParent(arguments);
         this.drawFeature();
     },
+
     /**
      * After rendering we setup our own custom events using #initCustomEvents.
      *
      * @private
      */
-    afterRender: function() {
+    afterRender: function () {
         this.callParent(arguments);
         this.initCustomEvents();
     },
+
     /**
      * (Re-)Initializes our custom event listeners, mainly #onClick.
      *
      * @private
      */
-    initCustomEvents: function() {
+    initCustomEvents: function () {
         this.clearCustomEvents();
         this.el.on("click", this.onClick, this);
     },
+
     /**
      * Unbinds previously bound listeners on #el.
      *
      * @private
      */
-    clearCustomEvents: function() {
+    clearCustomEvents: function () {
         if (this.el && this.el.clearListeners) {
             this.el.clearListeners();
         }
     },
+
     /**
      * Bound to the click event on the #el, this fires the click event.
      *
      * @private
      */
-    onClick: function() {
+    onClick: function () {
         this.fireEvent("click", this);
     },
+
     /**
      * Private method called during the destroy sequence.
      *
      * @private
      */
-    beforeDestroy: function() {
+    beforeDestroy: function () {
         this.clearCustomEvents();
         if (this.map) {
             this.map.setTarget(null);
         }
     },
+
     /**
      * When resizing has happened, we might need to re-set the renderer's
      * dimensions via #setRendererDimensions.
      *
      * @private
      */
-    onResize: function() {
+    onResize: function () {
         this.setRendererDimensions();
         this.callParent(arguments);
     },
+
     /**
      * Draw the feature in the map.
      *
      * @private
      */
-    drawFeature: function() {
+    drawFeature: function () {
         this.map.setTarget(this.el.id);
         this.setRendererDimensions();
     },
+
     /**
      * Set the dimension of our renderer, i.e. map and view.
      *
      * @private
      */
-    setRendererDimensions: function() {
+    setRendererDimensions: function () {
+        var resolution = this.initialConfig.resolution;
         var gb = this.feature.getGeometry().getExtent();
         var gw = ol.extent.getWidth(gb);
         var gh = ol.extent.getHeight(gb);
+        var width;
```

Comment:
This can also be a big PITA, in my experience. I have been through this forth and back and have come to the conclusion that I can live with the majority vote.